packages/data/src/atomic-store/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import { mapValues } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { createDerivedAtom } from '@wordpress/stan';
+
+/**
+ *
+ * @param {string}                                     name   Store name.
+ * @param {import('../types').WPDataAtomicStoreConfig} config Atomic store config.
+ * @return {import('../types').WPDataStore} Store.
+ */
+export default function createAtomicStore( name, config ) {
+	return {
+		name,
+		instantiate: ( registry ) => {
+			// I'm probably missing the atom resolver here
+			const selectors = mapValues( config.selectors, ( atomSelector ) => {
+				return ( /** @type {any[]} **/ ...args ) => {
+					const get = registry.__internalGetAtomResolver()
+						? registry.__internalGetAtomResolver()
+						: (
+								/** @type {import('@wordpress/stan/src/types').WPAtom<any>|import('@wordpress/stan/src/types').WPAtomFamilyItem<any>} **/ atom
```

Comment:
@sirreal, would know best, but I believe there is a way to avoid using `src/types` part.